webgui-new/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "webgui-new",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build && next export",
+    "start": "next start",
+    "lint": "next lint",
+    "thrift-ts": "thrift-typescript --fallbackNamespace none"
+  },
+  "devDependencies": {
+    "@creditkarma/thrift-typescript": "^3.7.6",
+    "@types/react": "18.0.26",
+    "@types/react-dom": "18.0.10",
+    "@types/thrift": "^0.10.12",
+    "dotenv": "^16.0.3",
+    "eslint": "8.31.0",
+    "eslint-config-next": "13.1.1",
+    "typescript": "4.9.4"
+  },
+  "dependencies": {
+    "@codemirror/lang-cpp": "^6.0.2",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.11.0",
+    "@mui/lab": "^5.0.0-alpha.116",
```

Comment:
An alpha version dependency seems suspicious.

Reply:
The @mui/lab package is part of the Material UI library, and is used to add more components to the main MUI package. From this package, I'm using the TreeView component, which is used to display the parts of the UI that need a tree-style view (for instance the file tree), so it is mandatory to use in the project. Because it's in alpha version, I carefully checked and it does not cause any problems and is safe to use in this case.

Furthermore, the package has constant maintenance and an active community, so it can be updated when it is out of the alpha version. Also, it is one of the most popular UI libraries, so I think we can make an exception in this case.
https://snyk.io/advisor/npm-package/@mui/lab